cmd/basecoin/commands/init.go

```diff
@@ -64,7 +67,24 @@ func initCmd(cmd *cobra.Command, args []string) error {
 		return errors.New("Address must be 20-bytes in hex")
 	}
 
-	genesis := GetGenesisJSON(viper.GetString(FlagChainID), userAddr)
+	var options []string
+	var optionsStr string
+	sep := ",\n      "
+	optionsRaw := viper.GetStringSlice(FlagOption)
+	if len(optionsRaw) > 0 {
+		optionsStr = sep
```

Comment:
Do we need this assignment? 

How about:

```
sed += strings.Join(option[:], sed)
```

Maybe we should rename sed then :-)

Reply:
Are you referring to L75? yes we need that because only if options are set to we want to include a (comma and new line) seperator